### PR TITLE
LPS-160362 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-160362

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/KBTemplate.macro [hunk 1]
```diff
@@ -25,6 +25,25 @@ definition {
...
+			key_itemName = "${kbTemplateTitle}",
+			locator1 = "KnowledgeBase#PAGE_TREE_ITEM_ELLIPSIS");
+
+		WaitForLiferayEvent.initializeLiferayEventLog();
+
+		if ("${menuItem}" == "Delete") {
+			MenuItem.clickNoError(menuItem = "${menuItem}");
+		}
+		else {
+			MenuItem.click(menuItem = "${menuItem}");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/KBTemplate.macro:85]: Is this conditional necessary?

Comment (Jeremy-Ch) [reply portal-web/test/functional/com/liferay/portalweb/macros/KBTemplate.macro:85]: Yeah, We need to click below confirmation info when we delete an entry. So we can only use `MenuItem.clickNoError`.
image

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**